README.md

```diff
@@ -57,14 +64,15 @@ r.word_of_the_day()
     - `maxDictionaryCount (integer)` - Maximum dictionary count (optional)
     - `minLength (integer)` - Minimum word length (optional)
     - `maxLength (integer)` - Maximum word length (optional)
+    - `length (integer)` - Exact word length (optional)
```

Comment:
This parameter doesn't comply with API spec provided by wordnik API. I think if any dev wanted to get specific word by length, then one can use `minLength = maxLength` while calling the API.

Plus, I am planning to extend this library by providing various sources, so we can add this optional parameter if providing random word using local dictionary. I still have to do research how, we can pull word from local dictionary present in OS though

Reply:
I understand that the Wordnik API doesn't support the parameter length. But as mentioned in Issue #22 by Erik Boesen, the function call for a word of a specific length is verbose and inefficient. So, what I've done is, I've accepted the optional parameter length from the user, set the max_length and min_length parameters internally to the value given in the parameter by the user and then deleted the length parameter from the kwargs before passing it to the other functions for exception handling. So basically, my change reduce unnecessary verbosity at the user end because otherwise if I wanted to get a 5 letter word, my call would have to be this

`rw.get_random_word(min_length=5, max_length=5)`

Now, from my commit, it would change to

`rw.get_random_word(length=5)`

This is both logical and more efficient for the user calling the function.